deployment-oauth.yaml

```diff
@@ -0,0 +1,260 @@
+apiVersion: kafka.strimzi.io/v1alpha1
```

Comment:
v1beta1?

Reply:
This is what we have also in all other examples. It has also the big advatnage that it works with older versions when you need to try them because of some questions. So I would keep this.

Reply:
but we are already using v1beta1 for KafkaUser here

Reply:
I guess I copied it from other place. I changed it to `v1alpha1`. If we want to change them all to `v1beta1` we should probably use different PR as it is much more files than just this one.

Reply:
Agree